frontend/packages/kubevirt-plugin/src/k8s/wrapper/common/object-with-type-property-wrapper.ts

```diff
@@ -22,6 +22,9 @@ export abstract class ObjectWithTypePropertyWrapper<
     super(data, copy);
     this.TypeClass = typeClass;
     this.typeDataPath = typeDataPath;
+    if (!_.isFunction(typeClass.getAll) || typeClass.getAll === ObjectEnum.getAll) {
```

Comment:
Wait, isn't `getAll` in `ObjectEnum` marked as `abstract` ?

Reply:
Because it is a static method.

Reply:
cannot be made abstract

Reply:
Oh, right.